jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/RemoteEndpoint.java

```diff
@@ -104,7 +104,7 @@
      * @param isLast true if this is the last piece of the partial bytes
      * @param callback callback to notify of success or failure of the write operation
      */
-    void sendPartialString(String fragment, boolean isLast, WriteCallback callback) throws IOException;
+    void sendPartialString(String fragment, boolean isLast, WriteCallback callback);
```

Comment:
This was wrong to have the `throws IOException` as this takes a callback, but I'm not sure if there will be any issues with changing it now that its already there.

Reply:
@lachlan-roberts no that will break applications that wrapped `sendPartialString()` into a `try/catch`.
You have to put `throws IOException` back. Perhaps add javadocs that say that it was a mistake and it's never thrown.